client/stylesheets/_modal.scss

```diff
@@ -654,11 +654,223 @@
 	font-size: 1rem !important; 
 }
 
+/* Hide Colors */
+.sp-palette-row-0 span:nth-child(2), .sp-palette-row-0 span:nth-child(4), .sp-palette-row-0 span:nth-child(5), .sp-palette-row-1 > span:first-child, .sp-palette-row-1 span:nth-child(7) {
+	display: none !important;
+}
+
+.froala-view {
+	height:180px !important;
+}
+
+.sp-palette, .sp-thumb-dark, .sp-thumb-dark.sp-thumb-active, .sp-thumb-inner, .sp-thumb-active {
+	background-image: none !important;
+}
+
+.sp-palette > div {
+	display: inline-block;
+}
+
+.sp-thumb-el {
+	width: 22px !important;
+	height:22px !important
+}
+
+.sp-container {
+	border: 1px solid #404040 !important;
+	top: 521.797px !important;
+}
+
+.sp-replacer {
+	border: 1px solid #404040 !important;
+}
+
+div[class="sp-replacer sp-light full-spectrum sp-active"] {
+	border-bottom: none !important;
+}
+
+.sp-dd {
+	display: none;
+}
+
+.sp-preview {
+	margin-right: 0 !important;
+	border: 1px solid #999 !important;
+}
+
+.sp-thumb-el.sp-thumb-active {
+	border-color: #999 !important;
+}
+
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	END END END END END END
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	Feedback Modal
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+
+/*~~~~ Button ~~~~*/
+
+.modFeedbackTrigger {
+
+}
+
+/*~~ Button End ~~*/
+
+/* Initial */
+
+.modFeedbackInitialPageOne, .modFeedbackInitialPageThree {
+	text-align: center;
+	margin-top: 20px;
+}
+
+.modFeedbackInitialPageOne h2, .modFeedbackInitialPageThree h2 {
```

Comment:
Selector `modFeedbackInitialPageOne` should be written in lowercase with hyphens<br>Selector `modFeedbackInitialPageThree` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own line